modules/core/tests/client/socket.io.client.service.tests.js

```diff
@@ -1,24 +1,39 @@
-(function() {
+(function(window) {
```

Comment:
https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md will have all the rules we pretty much follow.